lighthouse-core/audits/csp-xss.js

```diff
@@ -98,7 +100,7 @@ class CspXss extends Audit {
       const items = syntaxFindings[i].map(f => this.findingToTableItem(f));
       if (!items.length) continue;
       results.push({
-        severity: 'Syntax',
```

Comment:
I kind of like the idea that we continue to distinguish syntax errors from high severity findings.  

admittedly that does mean we'd have `Syntax Error` as a value under `Severity`, which... feels a little weird, but at the same time I think it works.

Reply:
I like calling this out 👍